### ISSUE TITLE
Server and Android port

### ISSUE THREAD

**Issue by jonsmirl** [christf/snapcastc#36]
I've been looking at the code for a few days and have hit a structural problem. The client and sever code are intermingled. This is a big problem for Android, because Android does not have alsa-lib. How hard is this to split out?

**Comment by christf** [comment]
I do not think it should be hard to split. Indeed it would be good to re-factor the relevant parts just as you are describing.

**Comment by christf** [comment]
with d03121c50de46c1492d566a0f3a94f35b4cd3254 the split is done.

**Comment by christf** [comment]
@jonsmirl where can I find you android port?
